Monika After Story/game/script-topics.rpy

```diff
@@ -15694,6 +15696,134 @@ label monika_eating_meat:
     m 3eua "Whatever we eat, the most important thing to me is that we try to put a little thought into where our food comes from."
     return
 
+# is player an introvert ?
+default persistent._mas_pm_introvert = None
+
+# is player an extrovert ?
+default persistent._mas_pm_extrovert = None
+
+# is player in-between ?
+default persistent._mas_pm_ambivert = None
+
+# is player unsure about where they fit on the spectrum ?
+default persistent._mas_pm_social_personality_unsure = None
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_introversion_extroversion",
+            prompt="Introverts and extroverts",
+            category=['psychology', 'society'],
+            action=EV_ACT_RANDOM,
```

Comment:
Please keep `conditional` and `action` grouped together. `aff_range` is best as the last property as it's cleaner to see all in all